README.md

```diff
@@ -51,55 +56,23 @@ npm install
 ```
 
-Now that the distribution is built you can load it up by serving the index.html file found in the package root. To serve it up from IntelliJ you can "Open in Browser". This will let you explore the API as it is exposed via the `LABKEY` global namespace.
+Now that the distribution is built you can load it up by serving the index.html file found in the package root. To 
+serve it up from IntelliJ you can "Open in Browser". This will let you explore the API as it is exposed via 
+the `LABKEY` global namespace.
 
 ### Running against a LabKey Server
 
-If you would like to experiment with running this code against LabKey Server it is possible thanks to the provided wrapper for the `LABKEY` global namespace. The following assumes you have a [LabKey development environment setup](https://www.labkey.org/Documentation/wiki-page.view?name=devMachine).
+If you would like to experiment with running this code against LabKey Server it is possible thanks to the provided 
+wrapper for the `LABKEY` global namespace. The following assumes you have a 
+[LabKey development environment setup](https://www.labkey.org/Documentation/wiki-page.view?name=devMachine).
 
-#### Deploy Using Experimental Build
+#### Deploy Using Experimental Flag
 
-:warning: This will modify files in your git enlistment. Double check your commits and be sure to revert any changes when you are done experimenting!
+Starting in version LabKey v20.5 this package can be used in LabKey Server by enabling 
+the "Use @labkey/api on the client-side" experimental feature.
 
-To get started edit the `<labkey root>/server/modules/core/build.js` file and set `USE_LABKEY_API` to `true`.
-
-```js
-const USE_LABKEY_API = true;
-```
-
-Next, perform a Gradle build from the top of your LabKey enlistment:
-
-```sh
-# From <labkey root>
-./gradlew deployApp
-```
-
-Once the build is complete, you can start the server and the code for the APIs will be supplied from this package.
-This can be verified by opening the browser console and typing:
-
-```js
-LABKEY.__package__
-```
-
-If installed correctly, `__package__` will contain package information such as the version of the package you're using.
-
-#### Deploy From Source
-
-Steps:
-1. Navigate to `<labkey root>/server/modules/platform/api/webapp/` and clone this repository.
-2. Navigate to the package directory and run `npm install` followed by `npm run build`.
-3. Open `<labkey root>/server/modules/platform/api/webapp/clientapi_core.lib.xml` and replace the contents
-
-```xml
-<libraries xmlns="http://labkey.org/clientLibrary/xml/">
-    <library compileInProductionMode="false">
-        <script path="labkey-api-js/dist/labkey-api-js-core.min.js"/>
-    </library>
-</libraries>
-```
-
-Now you can start the server and the code for the APIs will be supplied from this package. This can be verified
-by opening the browser console and typing:
+This flag requires either restarting the server or clearing the server caches to take effect. You can verify the
```

Comment:
I thought you solved the server cache issue?

Reply:
Good catch. Yup, I'd updated this README prior to fixing the caching issue. Fixed with 7c16d65.